testplans/docker-images/Dockerfile.oni-runtime-debug

```diff
@@ -10,21 +10,21 @@ FROM golang:${GO_VERSION}-buster as downloader
 
 RUN go get github.com/filecoin-project/go-paramfetch/paramfetch
 COPY /proof-parameters.json /
-RUN paramfetch 2048 /proof-parameters.json
+RUN paramfetch 8388608 /proof-parameters.json
 
-ARG LOTUS_COMMIT=1a170e18a
+ARG LOTUS_COMMIT=b13226bc2f8a0233b3beb9bd18ab266547bc29c7
```

Comment:
This is necessary for the `debug` image as we bundle `lotus-miner` and `lotus` binaries, so that we can inspect miners and clients while tests are running. This is NOT the version under test, but if it gets old enough, `lotus` and `lotus-miner` might stop working (due to CBOR serialisation changes for example).

Reply:
(also to the others)

Reply:
> Would it be worth rescuing the change log from README-old-from-oni.md? To keep tabs on the changes we're making.

I'd rather lock these tests with a specific Lotus commit. The initial thinking that "filecoin-ffi" does not change often is just not correct at this stage, we have updated it tens of times, it should be regarded as any other dependency in the `go.mod` of Lotus in my opinion.